src/seq_mv/csr_matop_device.c

```diff
@@ -11,6 +11,7 @@
  *
  *****************************************************************************/
 
+#include "_hypre_onedpl.hpp"
```

Comment:
It's hard for me to follow the changes in this file. Other than reordering things, name changing from CUDA to GPU, and adding new sycl code, any changes to the existing CUDA code?

Reply:
Again, should be no changes to existing code.